MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+    }
+  
+    public IEnumerable<OrderItem> GetItems()
+    {
+        // Changing the database table items as ObservableCollection
+        var table = (from i in database.Table<OrderItem>() select i);
+        ObservableCollection<OrderItem> OrderList = new ObservableCollection<OrderItem>();
+        foreach (var order in table)
+        {
+            OrderList.Add(new OrderItem()
+            {
+                ID = order.ID,
+                Name = order.Name
+            });
+        }
+        return OrderList;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Refer to the following code which illustrates, how to bind the data from the `SQLite` database to `SfListView`.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.SfListView">
+    <ContentPage.BindingContext>
+        <local:ViewModel/>
+    </ContentPage.BindingContext>
+    <ContentPage.Content>
+        <Grid>
+            <listView:SfListView x:Name="listView" ItemSize="70" 
+                                BackgroundColor="Teal"
+                        ItemsSource="{Binding OrderItemCollection}">
+                <listView:SfListView.ItemTemplate>
+                    <DataTemplate>
+                        <ViewCell>
+                            <ViewCell.View>
+                                <Grid>
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="*" />
+                                        <RowDefinition Height="1" />
+                                    </Grid.RowDefinitions>
+                                    <Grid>
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="*" />
+                                            <ColumnDefinition Width="*" />
+                                        </Grid.ColumnDefinitions>
+                                        <Label LineBreakMode="NoWrap" Text="{Binding ID}" VerticalOptions="Center"/>
+                                        <Label LineBreakMode="NoWrap" Text="{Binding Name}" Grid.Column="1" VerticalOptions="Center" />
+                                    </Grid>
+                                    <StackLayout Grid.Row="1" BackgroundColor="Gray" HeightRequest="1"/>
+                                </Grid>
+                            </ViewCell.View>
+                        </ViewCell>
+                    </DataTemplate>
+                </listView:SfListView.ItemTemplate>
+            </listView:SfListView>
+        </Grid>
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+{% endtabs %}
+
+## ListView with Prism Framework
+
+The `SfListView` allows the user to work with prism for MVVM Framework. Steps to be followed:
+
+1. Replace your application to prism application in App.xaml file.

```

Comment:
Replace your application with the prism application in the App.xaml file.

Reply:
I have modified the line as per the concerns provided